flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/source/Utils.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connector.jdbc.source;
+
+import org.apache.flink.connector.jdbc.source.split.CheckpointedOffset;
+import org.apache.flink.connector.jdbc.source.split.JdbcSourceSplit;
+import org.apache.flink.util.InstantiationUtil;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.Serializable;
+
+/** Utils class to hold common static methods. */
+public class Utils {
+    private Utils() {}
+
+    public static void serializeJdbcSourceSplit(DataOutputStream out, JdbcSourceSplit sourceSplit)
```

Comment:
I would say that this should be on JdbcSourceSplitSerializer.java, and we should instantiate that on JdbcSourceEnumStateSerializer.
This because any change to this code will affect the state and should be versioned, and is more easy to understand if is there.

Other option is to have this on JdbcSourceSplit, but I think that the same problem of versioning will happen.

But we should avoid this kind of utility.